app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,11 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href="" 
+                ng-attr-aria-disabled="{{canDeploy() ? undefined : 'true'}}"
+                ng-class="{ 'disabled-link': !canDeploy() }"
```

Comment:
I can do something like this:

```html
            <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
              <a
                href=""
                ng-if="canDeploy()"
                ng-click="row.startDeployment()">Deploy</a>
              <a
                href=""
                ng-if="!(canDeploy())"
                ng-attr-aria-disabled="{{canDeploy() ? undefined : 'true'}}"
                ng-class="{ 'disabled-link': !canDeploy() }">Deploy (paused)</a>
            </li>
```

![screen shot 2017-05-15 at 12 53 50 pm](https://cloud.githubusercontent.com/assets/280512/26069029/975e8dd8-396d-11e7-8ae6-8e7ca5492302.png)